modules/jooby-jedis/src/main/resources/org/jooby/jedis/jedis.conf

```diff
@@ -24,3 +24,7 @@ jedis.pool.jmxNamePrefix = redis-pool
 # session store, key prefix and timeout in seconds
 jedis.session.prefix = sessions
 jedis.session.timeout = ${session.timeout}
+
+jedis.sentinel.hosts = ["localhost:26379"]
```

Comment:
don't think we should have default values for all these variables

Reply:
For Redis Sentinel doesn't use these settings

> jedis.session.prefix = sessions
> jedis.session.timeout = ${session.timeout}

About this setting
> jedis.sentinel.hosts = ["localhost:26379"]

We can make it empty and check in constructor.

Reply:
yea, just want to avoid a default connection or connection where developer didn't notice. In order to connect they must provide the correct settings

Reply:
I've did it.